knowledge-base/how-to-create-a-dynamic-table-crosstab-list-item.md

```diff
@@ -48,7 +48,7 @@ Then, you can add a Table/List/Crosstab item with dynamic row and column groups
 
 ## Notes
 
-- The attached demo report illustrates the settings. The report can be previewed in the [Standalone Report Designer](../standalone-report-designer) of Telerik Reporting R1 2017 and newer.
+- The attached demo report illustrates the settings. The report can be previewed in the [Standalone Report Designer](../standalone-report-designer) of Telerik Reporting R1 2017 and later.
```

Comment:
the new link to the standalone designer is:
/designing-reports/report-designer-tools/desktop-designers/standalone-report-designer/overview

The old link is redirected, but it may be better to change it and use {slug}